backend/tests/Feature/Schedule/PlannerMaintenanceTest.php

```
@@ -0,0 +1,96 @@
+    }
+
+    public function test_line_and_a_maintenance_are_required(): void
+    {
+        $this->actingAs($this->admin)->post(route('admin.schedule.maintenance.store'), [
+            'scheduled_at' => now()->addDay()->format('Y-m-d H:i'),
+        ])->assertSessionHasErrors(['line_id', 'title']);
+    }
+
+    public function test_operator_cannot_place_maintenance(): void
+    {
+        $line = Line::factory()->create();
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Line::factory().


(staticMethod.notFound)